ml-prims/src/hmm/cuda/determinant.h

```diff
@@ -0,0 +1,128 @@
+// #pragma once
```

Comment:
Do we need this file if it is fully commented out?

Reply:
You are right, it's only needed later for the GMM PR